minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
Could have the following (this allows setter to be more strict about its input).

```java
   public void clearExistingZookeepers()
  /**
   * throws IllegalArgExcep if null or empty string
   */
   public void setExistingZookeepers(String ezk);
   public Optional<String> getExistingZookeepers();
```